charts/reporting-operator/templates/custom-resources/prom-queries/pvc-capacity.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: metering.openshift.io/v1alpha1
+kind: ReportPrometheusQuery
+metadata:
+  name: pvc-capacity-bytes
```

Comment:
Can you use `persistentvolumeclaim-` as the prefix in the name for consistency with how we name the other queries?